docs/integrations/sources/hubspot.md

```diff
@@ -1,14 +1,43 @@
 # HubSpot
 
-This page guides you through setting up the HubSpot source connector.
+This page contains the setup guide and reference information for the [HubSpot](https://www.hubspot.com/) source connector.
 
-## Prerequisite
+## Prerequisites
 
-You can use OAuth or a Private App to authenticate your HubSpot account.
+You can either use OAuth or set up a Private App to authenticate your HubSpot account.
 
-In Airbyte Cloud, we highly recommend you use OAuth and not Private App authentication as it significantly simplifies the setup process.
+For Airbyte Cloud users, we highly recommend you use OAuth rather than Private App authentication, as it significantly simplifies the setup process. Conversely, for Airbyte Open Source users we recommend Private App authentication.
```

Comment:
just making it visually clearer for anyone quickly scanning the page

```suggestion
**For Airbyte Cloud** users, we highly recommend you use OAuth rather than Private App authentication, as it significantly simplifies the setup process. 

**For Airbyte Open Source** users we recommend Private App authentication.
```

Reply:
Done!